Sources/CoreMetrics/Metrics.swift

```diff
@@ -12,23 +12,27 @@
 //
 //===----------------------------------------------------------------------===//
 
+// MARK: Testing API
+
+internal var _enableAssertions = true
+
 // MARK: User API
 
-extension Counter {
+public extension Counter {
```

Comment:
these formatting changes are not desired. we prefer to make the methods public not the extension. what version of swift-format did you use?

Reply:
`0.48.4`

Reply:
sorry about that :/ can you revert these and use 0.44.6? one swiftPM plugins are there, this kind of issues will go away hopefully.

Reply:
or if you want to keep the local version - run it via the docker setup

Reply:
done, ran with `0.44.6`